lib/facility_maker.rb

```diff
@@ -0,0 +1,79 @@
+class FacilityMaker
+
+    attr_reader :new_facilities, :new_facility_data
+
+    def initialize(dmv_data)
+        @dmv_data = dmv_data
+        @new_facilities = []
+        @new_facility_data = {}
+    end
+
+    def facility_maker_co
+        if @dmv_data.first[:state] == "CO"
+            data_organizer_co
+        end
+    end    
+
+    def data_organizer_co
+        @dmv_data.each do |location_data|
+            @new_facility_data[:name] = location_data[:dmv_office]
```

Comment:
I would just keep this as a local variable. We only need instance variables if it's going to be stored data and other methods will use that data. So I like `@new_facilities` as an ivar since that's data we'll want to hold onto, but we don't need `@new_facility_data` as an ivar, that's just local to this method.